doc/test_myst_doc.py

```diff
@@ -5,6 +5,7 @@
 import argparse
```

Comment:
This file was added from https://github.com/ray-project/ray/commit/c74886a55e81dc8f4150094bfec69d875f0b365d to fix CI test for notebooks.